js/main.js

```diff
@@ -107,107 +130,69 @@ document.addEventListener('DOMContentLoaded', function () {
 });
 
 // switch to show or hide no entry text
-const $noEntries = document.querySelector('.center');
+const $noEntries = document.querySelector('#center');
 function toggleNoEntries() {
-  if (data.entries.length > 0) {
-    $noEntries.setAttribute('class', 'hidden center');
+  if (data.entries.length < 1) {
+    $noEntries.className = 'center';
   } else {
-    $noEntries.setAttribute('class', 'center');
+    $noEntries.className = 'hidden';
   }
 }
 
 // viewswapping between entries and entry-form pages
 function viewSwap(viewName) {
   if (viewName === 'entry-form') {
-    $entries.setAttribute('class', 'hidden');
-    $entryForm.setAttribute('class', 'display');
+    $entries.classList.add('hidden');
+    $entryForm.classList.remove('hidden');
     data.view = viewName;
-    toggleNoEntries();
-  } else {
-    $entries.setAttribute('class', 'display');
-    $entryForm.setAttribute('class', 'hidden');
+  } else if (viewName === 'entries') {
+    $entryForm.classList.add('hidden');
+    $entries.classList.remove('hidden');
     data.view = viewName;
-    toggleNoEntries();
   }
 }
 
 // shows Entries page when the Entries tab is clicked
 $nav.addEventListener('click', function () {
+  data.view = 'entries';
```

Comment:
You shouldn't be setting `data.view` here, you should be setting this inside of `viewSwap`